packages/browser-repl/src/components/shell-input.tsx

```diff
@@ -110,10 +110,12 @@ export class ShellInput extends Component<ShellInputProps, ShellInputState> {
 
   private onEnter = async (): Promise<void> => {
     if (this.props.onInput) {
-      await this.props.onInput(this.state.currentValue);
+      const value = this.state.currentValue;
+      // clear the value before evaluating the input because it could take a
+      // long time
+      this.setState({ currentValue: '' });
```

Comment:
drive-by fix

Reply:
I suppose. A shell-input without onInput isn't going to be useful, but we might as well keep it sane.